hercules-ci-api/src/Hercules/API/Agent/Meta.hs

```diff
@@ -0,0 +1,44 @@
+{-# LANGUAGE DataKinds #-}
+{-# OPTIONS_GHC -fno-warn-deprecations #-}
+module Hercules.API.Agent.Meta where
```

Comment:
Maybe call it `LifeCycle`? Meta is too meta

Reply:
Why didn't I think of this.